### PR TITLE
Remove m_ prefix from most of the classes in 'games.strategy.common'

### PR DESCRIPTION
**Pull request by DanVanAtta**
This change is [Reviewable](https://reviewable.io/reviews/triplea-game/triplea/583)

### CODE REVIEW

File: src/games/strategy/common/ui/PlayersPanel.java [hunk 1]
```diff
@@ -19,17 +19,17 @@
...
   private static final long serialVersionUID = -4283654829822141065L;
-  private final PlayerManager m_players;
+  private final PlayerManager players;
 
   public PlayersPanel(final PlayerManager players, final GameData data) {
-    m_players = players;
+    this.players = players;
     setLayout(new BoxLayout(this, BoxLayout.Y_AXIS));
-    for (final String player : m_players.getPlayers()) {
+    for (final String player : this.players.getPlayers()) {
```
Comment (ron-murhammer) [inline src/games/strategy/common/ui/PlayersPanel.java]: Can we remove the 'this' here?

File: src/games/strategy/common/ui/PlayersPanel.java [hunk 2]
```diff
@@ -19,17 +19,17 @@
...
       final PlayerID playerID = data.getPlayerList().getPlayerID(player);
       if (playerID.isAI()) {
         add(new JLabel(playerID.getWhoAmI().split(":")[1] + " is " + playerID.getName(), JLabel.RIGHT));
       } else {
-        add(new JLabel(m_players.getNode(player).getName() + " is " + playerID.getName(), JLabel.RIGHT));
+        add(new JLabel(this.players.getNode(player).getName() + " is " + playerID.getName(), JLabel.RIGHT));
```
Comment (ron-murhammer) [inline src/games/strategy/common/ui/PlayersPanel.java]: Can we remove the 'this' here?

**Comment by ron-murhammer** [comment]
@DanVanAtta Added a few comments.

File: src/games/strategy/common/ui/PlayersPanel.java (hunk 2 above)
Comment (DanVanAtta) [inline src/games/strategy/common/ui/PlayersPanel.java]: One step better, can just remove the class level variable completely.  1807748

**Pull request merged**